localstack/services/sts/models.py

```diff
@@ -0,0 +1,9 @@
+from localstack.services.stores import AccountRegionBundle, BaseStore, CrossRegionAttribute
+
+
+class STSStore(BaseStore):
+    # maps access key ids to tags for the session they belong to
+    session_tags: dict[str, dict[str, str]] = CrossRegionAttribute(default=dict)
```

Comment:
Creating our own store is probably a good hint that we might want to internalize STS

Reply:
Yeah, would love that level of control to it, currently a lot of the operations are mocked.

Reply:
This will at some point be extended anyway, as we need to include session policies as well.